docs/embed-files.md

```diff
@@ -65,9 +65,11 @@ Sometimes you don't want to embed a whole file. Maybe because you need just a fe
 ```
 
 In your code file you need to surround the fragment between `/// [demo]` lines (before and after the fragment).
-Alternatively you can use `### [demo]`. By default, only identifiers are omitted. To omit the entire line containing the identifier in the fragment output, you can add the `:omitFragmentLine` option.
+Alternatively you can use `### [demo]`. By default, only identifiers are omitted. To omit the entire line containing the identifier in the fragment output, add the `:omitFragmentLine` option. This is useful if your code fragment is e.g. HTML and you want to hide the Docsify fragment identifier from showing in your HTML source. `<!-- /// [demo] -->` in your source file and `:omitFragmentLine` will make the `-->` not show up in your Docsify code fragment section.
 
-Example:
+Example ([\_media/example.js](/_media/example.js ':ignore')):
```

Comment:
Just checked in preview, this embed does not work since Vercel adds the trailing `preview` to the URL. Works locally though. Should probably be solved in a way that can handle both cases, maybe with a relative (`../`) path. Will look into it 

This is the URL for the file in preview, 
https://docsify-preview-p8gicu1k2-docsifyjs.vercel.app/preview/_media/example.js
while the link ATM links to this
https://docsify-preview-p8gicu1k2-docsifyjs.vercel.app/_media/example.js

Reply:
Seems to be fixed with just `_media/example.js` as the link (no leading slash)